docs/src/main/sphinx/connector/pushdown-char-varchar-case.fragment

```diff
@@ -0,0 +1,18 @@
+Predicate pushdown support
```

Comment:
Rename the fragments to make it obvious what their contents are.

I propose `no-inequality-pushdown-text-type` for this one. (better names welcome)

Reply:
Sounds great to me, done